<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Target</class>
 <widget class="QDialog" name="Target">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Files</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QLabel" name="description">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Select the folder the ilidata.xml should be stored (Target Folder) and your Toppingfiles (exported QML files etc.) should be written to (Subfolder). And add some meta information used for filenaming and in the ilidata.xml.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="projectname_label">
        <property name="text">
         <string>Project Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="projectname_line_edit"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="version_label">
        <property name="text">
         <string>Version</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QgsDateTimeEdit" name="version_date_edit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="displayFormat">
         <string>dd.MM.yyyy</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="owner_label">
        <property name="text">
         <string>Owner (URI)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="owner_line_edit"/>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="publishingdate_label">
        <property name="text">
         <string>Publishing Date</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QgsDateTimeEdit" name="publishingdate_date_edit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="displayFormat">
         <string>dd.MM.yyyy</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="sub_folder_line_edit"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="sub_folder_label">
        <property name="text">
         <string>Target Folder</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="main_folder_browse_button">
        <property name="toolTip">
         <string>Browse XTF files</string>
        </property>
        <property name="text">
         <string>â€¦</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="main_folder_label">
        <property name="text">
         <string>Subfolder</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="main_folder_line_edit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Structure Preview</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QTextBrowser" name="info_text_box">
     <property name="enabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsDateTimeEdit</class>
   <extends>QDateTimeEdit</extends>
   <header>qgsdatetimeedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>projectname_line_edit</tabstop>
  <tabstop>owner_line_edit</tabstop>
  <tabstop>version_date_edit</tabstop>
  <tabstop>publishingdate_date_edit</tabstop>
  <tabstop>main_folder_line_edit</tabstop>
  <tabstop>main_folder_browse_button</tabstop>
  <tabstop>sub_folder_line_edit</tabstop>
  <tabstop>info_text_box</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
